parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1796,12 +1796,27 @@ applySelection hqs file = \sr@SlurpResult{..} ->
 var :: Var v => Name -> v
 var name = Var.named (Name.toText name)
 
-toSlurpResult :: forall v. Var v => UF.TypecheckedUnisonFile v Ann -> Names0 -> SlurpResult v
-toSlurpResult uf existingNames =
-  Slurp.subtractComponent (conflicts <> ctorCollisions) $
-  SlurpResult uf mempty adds dups mempty conflicts updates
-              termCtorCollisions ctorTermCollisions termAliases typeAliases
-              mempty
+toSlurpResult
+  :: forall v
+   . Var v
+  => Path.Absolute
```

Comment:
Passing the current path so we can deduplicate names that have both absolute and relative versions in the `names`.